VAMobile/src/store/slices/errorSlice.ts

```diff
@@ -24,7 +17,6 @@ export type DowntimeWindowsByFeatureType = {
 }
 
 export type DowntimeWindow = {
-  featureName: string
```

Comment:
Removed this because it should be determined by screen, not by upstream feature name.